.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-      - main
+      - ci
```

Comment:
Sorry for not noticing this before.

I think this could always be `main`, but in addition to that 
```yaml
on:
  pull_request:
  push:
    branches:
      - main
 ```

So that it runs on either main or on pull requests